### PR TITLE
Add support for rpi-node and rpi-buildstep-deps

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: rpi-node/content.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+> [wikipedia.org/wiki/Node.js](https://en.wikipedia.org/wiki/Node.js)
+
+%%LOGO%%
+
+# How to use this image
+
+## Create a `Dockerfile` in your Node.js app project
+
+	FROM node:0.10-onbuild
```
Comment (petrosagg) [inline rpi-node/content.md]: this should be `resin/rpi-node`

File: rpi-node/content.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Notes
+
+The image assumes that your application has a file named [`package.json`](https://docs.npmjs.com/files/package.json) listing its dependencies and defining its [start script](https://docs.npmjs.com/misc/scripts#default-values).
+
+## Run a single Node.js script
+
+For many simple, single file projects, you may find it inconvenient to write a complete `Dockerfile`. In such cases, you can run a Node.js script by using the Node.js Docker image directly:
+
+	docker run -it --rm --name my-running-script -v "$PWD":/usr/src/myapp -w /usr/src/myapp node:0.10 node your-daemon-or-script.js
```
Comment (petrosagg) [inline rpi-node/content.md]: this should be `resin/rpi-node`

File: rpi-node/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Support for older versions (down to 1.0) is provided on a best-effort basis.
+
+# User Feedback
+
+## Issues
+
+If you have any problems with or questions about this image, please contact us through a [GitHub issue](https://github.com/resin-io-library/docker-node/issues).
+
+You can also reach many of the official image maintainers via the `#docker-library` IRC channel on [Freenode](https://freenode.net).
```
Comment (petrosagg) [inline rpi-node/README.md]: This has to go, we don't have an active IRC channel.

File: rpi-node/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+### Notes
+
+The image assumes that your application has a file named [`package.json`](https://docs.npmjs.com/files/package.json) listing its dependencies and defining its [start script](https://docs.npmjs.com/misc/scripts#default-values).
+
+## Run a single Node.js script
+
+For many simple, single file projects, you may find it inconvenient to write a complete `Dockerfile`. In such cases, you can run a Node.js script by using the Node.js Docker image directly:
+
+	docker run -it --rm --name my-running-script -v "$PWD":/usr/src/myapp -w /usr/src/myapp node:0.10 node your-daemon-or-script.js
```
Comment (petrosagg) [inline rpi-node/README.md]: this should be `resin/rpi-node`

File: rpi-node/README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+> [wikipedia.org/wiki/Node.js](https://en.wikipedia.org/wiki/Node.js)
+
+![logo](https://raw.githubusercontent.com/docker-library/docs/master/rpi-node/logo.png)
+
+# How to use this image
+
+## Create a `Dockerfile` in your Node.js app project
+
+	FROM node:0.10-onbuild
```
Comment (petrosagg) [inline rpi-node/README.md]: this should be `resin/rpi-node`

File: rpi-node/README.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+Node.js is a software platform for scalable server-side and networking applications. Node.js applications are written in JavaScript and can be run within the Node.js runtime on Mac OS X, Windows, and Linux without changes.
+
+Node.js applications are designed to maximize throughput and efficiency, using non-blocking I/O and asynchronous events. Node.js applications run single-threaded, although Node.js uses multiple threads for file and network events. Node.js is commonly used for real-time applications due to its asynchronous nature.
+
+Node.js internally uses the Google V8 JavaScript engine to execute code; a large percentage of the basic modules are written in JavaScript. Node.js contains a built-in, asynchronous I/O library for file, socket, and HTTP communication. The HTTP and socket support allows Node.js to act as a web server without additional software such as Apache.
+
+> [wikipedia.org/wiki/Node.js](https://en.wikipedia.org/wiki/Node.js)
+
+![logo](https://raw.githubusercontent.com/docker-library/docs/master/rpi-node/logo.png)
```
Comment (petrosagg) [inline rpi-node/README.md]: Wrong link here

File: rpi-node/README.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+-	[`0.12.0-wheezy`, `0.12-wheezy`, `0-wheezy`, `wheezy` (*0.12/wheezy/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/5bca320dbf01b25125a2785fe3aeddea644f57c4/0.12/wheezy/Dockerfile)
+-	[`0.8.28`, `0.8` (*0.8/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/0de2ce005deb23a5ec9eaa667c5f6128ec323f79/0.8/Dockerfile)
+-	[`0.8.28-onbuild`, `0.8-onbuild` (*0.8/onbuild/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/0de2ce005deb23a5ec9eaa667c5f6128ec323f79/0.8/onbuild/Dockerfile)
+-	[`0.8.28-slim`, `0.8-slim` (*0.8/slim/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/d7af46b394ccb3f4d9527cdd01e85b44edd2366b/0.8/slim/Dockerfile)
+-	[`0.8.28-wheezy`, `0.8-wheezy` (*0.8/wheezy/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/0a0c83fa341710b6a55a07c3eadc9a44a6bc84fe/0.8/wheezy/Dockerfile)
+
+For more information about this image and its history, please see the [relevant manifest file (`library/rpi-node`)](https://github.com/resin-io-library/official-images/blob/master/library/rpi-node) in the [`docker-library/official-images` GitHub repo](https://github.com/resin-io-library/official-images).
```
Comment (petrosagg) [inline rpi-node/README.md]: in the [`docker-library/official-images` GitHub repo] -> in the [`resin-library/official-images` GitHub repo]

File: rpi-node/README.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+-	[`0.10.36-wheezy`, `0.10-wheezy` (*0.10/wheezy/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/5bca320dbf01b25125a2785fe3aeddea644f57c4/0.10/wheezy/Dockerfile)
+-	[`0.12.0`, `0.12`, `0`, `latest` (*0.12/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/5bca320dbf01b25125a2785fe3aeddea644f57c4/0.12/Dockerfile)
+-	[`0.12.0-onbuild`, `0.12-onbuild`, `0-onbuild`, `onbuild` (*0.12/onbuild/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/0a0c83fa341710b6a55a07c3eadc9a44a6bc84fe/0.12/onbuild/Dockerfile)
+-	[`0.12.0-slim`, `0.12-slim`, `0-slim`, `slim` (*0.12/slim/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/5bca320dbf01b25125a2785fe3aeddea644f57c4/0.12/slim/Dockerfile)
+-	[`0.12.0-wheezy`, `0.12-wheezy`, `0-wheezy`, `wheezy` (*0.12/wheezy/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/5bca320dbf01b25125a2785fe3aeddea644f57c4/0.12/wheezy/Dockerfile)
+-	[`0.8.28`, `0.8` (*0.8/Dockerfile*)](https://github.com/resin-io-library/docker-node/blob/0de2ce005deb23a5ec9eaa667c5f6128ec323f79/0.8/Dockerfile)
```
Comment (petrosagg) [inline rpi-node/README.md:11]: Why are these at a different commit than the 0.10 ones?

File: rpi-buildpack-deps/README.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+Support for older versions (down to 1.0) is provided on a best-effort basis.
+
+# User Feedback
+
+## Issues
+
+If you have any problems with or questions about this image, please contact us through a [GitHub issue](https://github.com/resin-io-library/buildpack-deps/issues).
+
+You can also reach many of the official image maintainers via the `#docker-library` IRC channel on [Freenode](https://freenode.net).
```
Comment (petrosagg) [inline rpi-buildpack-deps/README.md]: This has to go

File: rpi-buildpack-deps/README.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+-	[`wheezy` (*wheezy/Dockerfile*)](https://github.com/resin-io-library/buildpack-deps/blob/550c14ae65dc2c3529167f1f6b7bf41ac5a8a626/wheezy/Dockerfile)
+
+For more information about this image and its history, please see the [relevant manifest file (`library/rpi-buildpack-deps`)](https://github.com/docker-library/official-images/blob/master/library/rpi-buildpack-deps) in the [`docker-library/official-images` GitHub repo](https://github.com/docker-library/official-images).
+
+# What is `buildpack-deps`?
+
+In spirit, `buildpack-deps` is similar to [Heroku's stack images](https://github.com/heroku/stack-images/blob/master/bin/cedar.sh). It includes a large number of "development header" packages needed by various things like Ruby Gems, PyPI modules, etc. For example, `buildpack-deps` would let you do a `bundle install` in an arbitrary application directory without knowing beforehand that `ssl.h` is required to build a dependent module.
+
+![logo](https://raw.githubusercontent.com/docker-library/docs/master/rpi-buildpack-deps/logo.png)
```
Comment (petrosagg) [inline rpi-buildpack-deps/README.md]: wrong link here

File: rpi-buildpack-deps/README.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+-	[`jessie-scm`, `scm` (*jessie/scm/Dockerfile*)](https://github.com/resin-io-library/buildpack-deps/blob/550c14ae65dc2c3529167f1f6b7bf41ac5a8a626/jessie/scm/Dockerfile)
+-	[`jessie`, `latest` (*jessie/Dockerfile*)](https://github.com/resin-io-library/buildpack-deps/blob/550c14ae65dc2c3529167f1f6b7bf41ac5a8a626/jessie/Dockerfile)
+-	[`wheezy-curl` (*wheezy/curl/Dockerfile*)](https://github.com/resin-io-library/buildpack-deps/blob/550c14ae65dc2c3529167f1f6b7bf41ac5a8a626/wheezy/curl/Dockerfile)
+-	[`wheezy-scm` (*wheezy/scm/Dockerfile*)](https://github.com/resin-io-library/buildpack-deps/blob/550c14ae65dc2c3529167f1f6b7bf41ac5a8a626/wheezy/scm/Dockerfile)
+-	[`wheezy` (*wheezy/Dockerfile*)](https://github.com/resin-io-library/buildpack-deps/blob/550c14ae65dc2c3529167f1f6b7bf41ac5a8a626/wheezy/Dockerfile)
+
+For more information about this image and its history, please see the [relevant manifest file (`library/rpi-buildpack-deps`)](https://github.com/docker-library/official-images/blob/master/library/rpi-buildpack-deps) in the [`docker-library/official-images` GitHub repo](https://github.com/docker-library/official-images).
```
Comment (petrosagg) [inline rpi-buildpack-deps/README.md]: docker-library -> resin-io-library

File: .template-helpers/template.md [hunk 12]
```diff
@@ -2,7 +2,7 @@
 
 %%TAGS%%
 
-For more information about this image and its history, please see the [relevant manifest file (`library/%%REPO%%`)](https://github.com/docker-library/official-images/blob/master/library/%%REPO%%) in the [`docker-library/official-images` GitHub repo](https://github.com/docker-library/official-images).
+For more information about this image and its history, please see the [relevant manifest file (`library/%%REPO%%`)](https://github.com/resin-io-library/official-images/blob/master/library/%%REPO%%) in the [`docker-library/official-images` GitHub repo](https://github.com/resin-io-library/official-images).
```
Comment (petrosagg) [inline .template-helpers/template.md]: docker-library -> resin-io-library

Comment (petrosagg) [inline .template-helpers/template.md]: library -> resin

File: rpi-node/README.md (hunk 8 above)
Comment (nghiant2710) [inline rpi-node/README.md:11]: There is a minor fix in Dockerfiles except for onbuild. That's why the commits are different.

**Comment by petrosagg** [comment]
Can you edit the commit message to something like "Add support for rpi-node and rpi-buildstep-deps" ?

**Pull request merged**